From b39e160d3974613328c570f46ecbcbcb70a5101b Mon Sep 17 00:00:00 2001
From: "U. Artie Eoff" <ullysses.a.eoff@intel.com>
Date: Tue, 12 May 2020 11:01:30 -0700
Subject: [PATCH] Fix HEVCe kernel compile error on gcc10

This fixes a "multiple definition" compiler error on
gcc10 which defaults to -fno-common.

https://gcc.gnu.org/gcc-10/porting_to.html#common

Fixes #503

Signed-off-by: U. Artie Eoff <ullysses.a.eoff@intel.com>
---
 src/gen9_hevc_enc_kernels_binary.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gen9_hevc_enc_kernels_binary.h b/src/gen9_hevc_enc_kernels_binary.h
index 29659fa..ff7d0af 100644
--- a/src/gen9_hevc_enc_kernels_binary.h
+++ b/src/gen9_hevc_enc_kernels_binary.h
@@ -31,6 +31,6 @@
 
 #define GEN9_HEVC_ENC_KERNEL_SIZE 149296
 
-const unsigned int gen9_hevc_encoder_kernels[GEN9_HEVC_ENC_KERNEL_SIZE];
+extern const unsigned int gen9_hevc_encoder_kernels[GEN9_HEVC_ENC_KERNEL_SIZE];
 
 #endif
-- 
2.16.4

